index.ts

```diff
@@ -6,6 +6,7 @@ export interface ReplicatorMetaInput {
         host: string
         project_api_key: string
         replication: string
+        eventsToIgnore: string
```

Comment:
Let's follow the snake_case convention here
```suggestion
        events_to_ignore: string
```